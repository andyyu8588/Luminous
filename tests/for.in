// For compiler/VM testing purpose

k = 1;
for (a from k to 10 by 2) {
  print(a);
}

{
    j = 15;
    for(a from k to j by 5) {
        print(a);
    }
    {
        i = -3;
        for(a from j to i by -2) {
            print(a);
        }
    }
}