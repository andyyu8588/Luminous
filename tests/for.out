1
3
5
7
9
1
6
11
15
13
11
9
7
5
3
1
-1
